src/components/Toolbar.tsx

```diff
@@ -155,85 +174,85 @@ const Toolbar = (): JSX.Element => {
         active={checkActiveElement('h1')}
         handler={(e) => toggleBlock(e, 'h1')}
       >
```

Comment:
Now that all these are images and not text, it would be nice with a title to at least have a bare minimum of accessibility.

```suggestion
        handler={(e) => toggleBlock(e, 'h1')}
        title="Heading 1"
      >
```

Would also require
```diff
interface ButtonProps extends React.PropsWithChildren {
  handler: (e: React.PointerEvent) => void;
  active?: boolean;
+ title?: string;
}
```
and
```diff
    <button
      className={cx('_legoEditor_Toolbar_button', className)}
      onPointerDown={(e) => handleClick(e)}
+     title={title}
      type="button"
    >
```
or you could add the title to the icon - whichever you prefer

Reply:
Good point! Most of these were already images, so this was actually never really supported. Added aria-labels to all now